### ISSUE TITLE
Natu is not open source / Free software

### ISSUE THREAD

**Issue by LourensVeen** [kdavies4/natu#36]
The natu license is a 3-clause BSD license with an additional term, which requires anyone downloading or using the library to comply with US export law. This is not compatible with clauses 1 and 5 of the [Open Source Definition](https://opensource.org/osd), [nor with the Free Software Definition](https://www.gnu.org/philosophy/free-sw.html#exportcontrol).

I'd like to ask you to consider removing this clause, as it makes combining natu with other software difficult enough that I'd recommend against using it, and that's a shame, because it looks quite good.

If that is impossible or undesirable, then please note clearly that natu is not Open Source or Free Software, and that it is not GPL compatible, in the documentation, e.g. under "License terms and development" on [this page](http://kdavies4.github.io/natu/). Also, the PyPI categories should be updated, to avoid misleading people into thinking that it's under a plain BSD license.

**Comment by kdavies4** [comment]
Thanks for your comment. That additional term was required by the Georgia Tech Research Institute due to export restrictions which existed at the time I developed the software as a grad student. It may be possible to remove it now, but unfortunately I don't have time to properly pursue it at this time. I will try to make the changes you suggested though.

**Comment by LourensVeen** [comment]
Yes, I can see how arguing with a university bureaucracy is not feasible. Thanks for making those clarifications, that helps a lot!

**Comment by johnflavin** [comment]
@kdavies4 What would be required to make this change, or to see if this change could be made?

This clause in the license was brought up in https://github.com/conda-forge/staged-recipes/pull/11075, where I tried making a conda recipe for this package, as a reason that this package might not be distributable through conda forge.

**Comment by kdavies4** [comment]
@johnflavin I'm sorry for the delay. I will contact GTRC which originally imposed the extra terms on the license.
